charts/drax/values.yaml

```diff
@@ -716,6 +727,45 @@ influxdb:
         CREATE DATABASE "db_5G" WITH DURATION 5d REPLICATION 1 NAME "rp_5d_5G"
 
 
+influxdb2:
+  enabled: true
+
+  podLabels:
+    drax/role: ric
+    drax/component-name: influxdb2
+
+  adminUser:
+    organization: "accelleran"
+    bucket: "default"
+    retention_policy: "0s"
+    user: "admin"
+    password: ""
+    token: "UcmOqrU5834fniAi6cRfCLa3OkxZrh9Y"
```

Comment:
```suggestion
    token: "accelleran"
```

Reply:
This was done in our interactive session